doc/howto.md

```diff
@@ -280,15 +280,15 @@ In order to do so, add the following require statement to the initialiation code
     request = require('request');
 ```
 
-and add the `request` dependency to the `package.json` file: 
+and add the `request` dependency to the `package.json` file:
 
 ```json
```

Comment:
This is similar to https://github.com/telefonicaid/lightweightm2m-iotagent/pull/130#discussion_r203048085

In this case, given this is a fragmetn of a JSON which doesn't striclty follows the JSON syntax I'd suggest don't add the highligth keyword.

Reply:
Fixed 3c19e67 - Removed Highlight